Makefile

```diff
@@ -170,6 +170,8 @@ check:
 	@! ineffassign . | grep -vF gossip/gossip.pb.go
 	@echo "errcheck"
 	@errcheck -ignore 'bytes:Write.*,io:Close,net:Close,net/http:Close,net/rpc:Close,os:Close,database/sql:Close' $(PKG)
+	@echo "returncheck"
+	@returncheck $(PKG)
```

Comment:
It's strange that this references `PKG`, but I see you are following the pattern of some of the other checks. Does anyone use `make check PKG=./foo`? I would change this and other checks to use `./...`. If someone wants to run `returncheck` on a specific package they can do `returncheck ./foo`.